main/settings.py

```diff
@@ -795,13 +795,15 @@ def get_all_config_keys():
     name="QDRANT_COLLECTION_NAME", default="resource_embeddings"
 )
 
-QDRANT_SEARCH_VECTOR_NAME = get_string(
-    name="QDRANT_SEARCH_VECTOR_NAME", default="fast-bge-small-en"
-)
 
-QDRANT_DENSE_MODEL = get_string(
-    name="QDRANT_DENSE_MODEL", default="sentence-transformers/all-MiniLM-L6-v2"
-)
+QDRANT_DENSE_MODEL = get_string(name="QDRANT_DENSE_MODEL", default=None)
 QDRANT_SPARSE_MODEL = get_string(
     name="QDRANT_SPARSE_MODEL", default="prithivida/Splade_PP_en_v1"
 )
+QDRANT_ENCODER = get_string(
+    name="QDRANT_ENCODER", default="vector_search.encoders.fastembed.FastEmbedEncoder"
+)
+OPENAI_EMBEDDINGS_API_KEY = get_string(
+    name="OPENAI_EMBEDDINGS_API_KEY",
```

Comment:
This may be a good discussion point. I currently have the embedding api key scoped for permissions to just embeddings. Do we want to have a catch all key or should we scope them?

Reply:
I swapped out the openai encoder for a litellm encoder which means we will have to use OPENAI_API_KEY as the key name and have to set that as an environment variable so that litellm picks it up. to test the new litellm encoder set the OPENAI_API_KEY environment variable and settings.QDRANT_ENCODER to the litellm encoder.